src/main/kotlin/no/nav/arbeidsgiver/min_side/varslingstatus/VarslingStatusRepository.kt

```diff
@@ -31,8 +31,10 @@ class VarslingStatusRepository(
         return namedParameterJdbcTemplate.queryForList(
             """
             select status, varslet_tidspunkt, status_tidspunkt
-                from varsling_status
+                from varsling_status vs
+                left join kontaktinfo_resultat ki using (virksomhetsnummer)
                 where virksomhetsnummer = :virksomhetsnummer
+                    and (ki is null or (ki.har_epost = false and ki.har_tlf = false))
```

Comment:
har ikke sett `ki is null` før. kult!

Reply:
var faktisk litt usikker på om det gikk, men det funker fjell i følge testene